src/components/pages/AllProjects.tsx

```diff
@@ -136,14 +140,29 @@ const AllProjects = () => {
         {viewMode === 'organizations' && (
           <RootPermissionsFilter permissions={['canEditOrganization']}>
             <Grid item xs={12}>
-              <ButtonLink
-                data-testid='addOrganizationButton'
-                to={generateSafePath(Routes.CREATE_ORGANIZATION)}
-                Icon={AddIcon}
-                leftAlign
-              >
-                Add Organization
-              </ButtonLink>
+              <Stack gap={2} direction={isMobile ? 'column' : 'row'}>
+                <CommonSearchInput
```

Comment:
This search only shows up if the user has permission to edit organizations, because of the `RootPermissionsFilter` wrapper. We should show it in all cases. The wrapper only needs to hide the Add Organization button